server-sdk/src/main/java/com/amazonaws/ml/mms/servingsdk/http/Request.java

```diff
@@ -0,0 +1,18 @@
+package com.amazonaws.ml.mms.servingsdk.http;
+
+import java.util.List;
+import java.util.Map;
+
+public interface Request {
+    List<String> getHeaderNames();
+    String getRequestURL();
```

Comment:
We don't need both URI and URL

Reply:
True. I didn't implement it yet. I will remove it for now.